src/input-range-select/input-range-select.tsx

```diff
@@ -291,6 +300,17 @@ export const InputRangeSelect = <T, V>({
                     {renderSelectorContent("from")}
                     {renderSelectorContent("to")}
                 </RangeInputInnerContainer>
+                {focusedInput === "none" &&
```

Comment:
missed this earlier, but there's a `validateDOMNesting(...): <button> cannot appear as a descendant of <button>.` warning in the console

since the Selector is already a button, the clear button has to be moved out and rendered outside of it

Reply:
noticed the addition of a `InputSelectorWrapper`, but the clear icon hasn't been moved so the warning is still present